### DISCUSSION TITLE
Community Review: Registry for Codecs

### DISCUSSION THREAD

**Discussion by Alt-Shivam** [zarr-developers/zarr-python#1055] [General]
Hii Zarr Community,
So, we have the initial version of Zarr Codecs documented in a research file. 🎉 
Have a Look. [Link](https://hackmd.io/@uTe8Vo8gSYeCbwHsQI2Z2Q/SJycGaHtq)

This research is basically to make a Registry of Codecs at the end.
This registry will serve as a critical bridge between several communities from different programming languages.
and will be in Human as well as machine-readable format.

So, community reviews and suggestions become really important. 💯 

Project Information: [Link](https://hackmd.io/@uTe8Vo8gSYeCbwHsQI2Z2Q/SypXtPRD9)
Research file: [Link](https://hackmd.io/@uTe8Vo8gSYeCbwHsQI2Z2Q/SJycGaHtq)

Will wait for your precious suggestions and reviews.
Thanks 😄 

cc: @joshmoore @MSanKeys963 @grlee77 @martindurant @benjeffery @WardF @martindurant @jakirkham @rabernat @alimanfoo @parthxtripathi and all the community members.

**Comment by martindurant** [comment]
A few are being developed as part of [`kerchunk`](https://github.com/fsspec/kerchunk).

Note that in the current release of numcodecs, it should not be necessary for users to register their codecs, they should be found from entrypoints information, if the package containing them has it.
